src/app/shared/components/display-table/display-table.component.html

```diff
@@ -1,69 +1,105 @@
+      <div class="value" [cdkCopyToClipboard]="report.description">{{report.description}}</div>
+    </div>
+    <div class='table-row' *ngIf='!report.xml'>
+      <div class="key">Thread name</div>
+      <div class="value" [cdkCopyToClipboard]="report.threadName">{{ report.threadName}}</div>
+    </div>
+    <div class='table-row' *ngIf='!report.xml && report.sourceClassName'>
+      <div class="key">Source class name</div>
+      <div class="value" [cdkCopyToClipboard]="report.sourceClassName">{{ report.sourceClassName}}</div>
+    </div>
+    <div class='table-row' *ngIf='!report.xml && report.messageClassName'>
+      <div class="key">Message class name</div>
+      <div class="value" [cdkCopyToClipboard]="report.messageClassName">{{ report.messageClassName}}</div>
+    </div>
+    <div class='table-row' *ngIf='!report.xml && report.path'>
+      <div class="key">Path</div>
+      <div class="value" [cdkCopyToClipboard]="report.path">{{ report.path}}</div>
+    </div>
+    <div class='table-row' *ngIf='!report.xml && report.uid'>
+      <div class="key">CheckpointUID</div>
+      <div class="value" [cdkCopyToClipboard]="report.uid">{{ report.uid}}</div>
+    </div>
+    <div class='table-row' *ngIf='!report.xml && report.level'>
+      <div class="key">Level</div>
+      <div class="value" [cdkCopyToClipboard]="report.level">{{ report.level}}</div>
+    </div>
+    <div class='table-row' *ngIf='!report.xml && report.encoding'>
+      <div class="key">Encoding</div>
+      <div class="value" [cdkCopyToClipboard]="report.encoding">{{ report.encoding}}</div>
+    </div>
+    <div class='table-row' *ngIf='!report.xml && report.message'>
+      <div class="key">Number of characters</div>
+      <div class="value" [cdkCopyToClipboard]="report.message.length">{{ report.message.length}}</div>
+    </div>
+    <div class='table-row' *ngIf='report.xml && report.transformation'>
+      <div class="key">Transformation</div>
+      <div class="value" [cdkCopyToClipboard]="report.transformation">{{ report.transformation}}</div>
+    </div>
+    <div class='table-row' *ngIf='report.xml && report.storageId'>
+      <div class="key">StorageId</div>
+      <div class="value" [cdkCopyToClipboard]="report.storageId">{{ report.storageId}}</div>
+    </div>
+    <div class='table-row' *ngIf='report.xml && report.storage'>
+      <div class="key">Storage</div>
+      <div class="value" [cdkCopyToClipboard]="report.storage">{{ report.storage}}</div>
+    </div>
+    <div class='table-row' *ngIf='report.xml && report.estimatedMemoryUsage'>
+      <div class="key">Estimated memory usage</div>
+      <div class="value" [cdkCopyToClipboard]="report.estimatedMemoryUsage">{{ report.estimatedMemoryUsage}}</div>
+    </div>
+    <div class='table-row' *ngIf='report.xml && report.correlationId'>
+      <div class="key">CorrelationId</div>
+      <div class="value" [cdkCopyToClipboard]="report.correlationId">{{ report.correlationId}}</div>
+    </div>
```

Comment:
Maybe an idea but cant you wrap this in an `ng-container` with an if / else check on `report.xml`, that way more readable and you dont have to do the check the same property on every condition?